InfiniTAM/infinitam_ros_node.cpp

```diff
@@ -222,6 +220,13 @@ bool InfinitamNode::startInfinitam(std_srvs::SetBool::Request& request,
     image_source_->main_engine_ = main_engine_;
     pose_source_->main_engine_ = main_engine_;
 
+    // TODO(gocarlos): it would be nice to output the name of the enum here
```

Comment:
Thanks for the suggestion. I looked at it but did not find a nice way to do that. 
Since this is only be a " would be nice " TODO I'll skip it...

http://stackoverflow.com/questions/5093460/how-to-convert-an-enum-type-variable-to-a-string